shell/initialize/index.js

```diff
@@ -242,6 +275,10 @@ async function createApp(ssrContext, config = {}) {
   }
 
   // Add enablePreview(previewData = {}) in context for plugins
+  // https://v2.nuxt.com/docs/features/live-preview/
+  // how relevant is this? Is this connected to SSR?
+  // is this tied to https://github.com/rancher/dashboard/blob/master/shell/components/ResourceYaml.vue#L82?
+  // - no need for this gating....
   if (process.static && process.client) {
```

Comment:
i think process.static should always be false (but we don't plumb it in, so is undefined. See https://github.com/rancher/dashboard/pull/9543#issuecomment-1700763405 for a bit more info